Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1532,4 +1536,98 @@ namespace Babylon
     {
         return Napi::Value::From(info.Env(), m_graphicsImpl.GetHardwareScalingLevel());
     }
+
+    Napi::Value NativeEngine::CreateImageBitmap(const Napi::CallbackInfo& info)
+    {
+        auto env = info.Env();
```

Comment:
Can this be `const` or no?

Reply:
Yes! Setting a few others as const as well.